app/src/androidTest/java/org/simple/clinic/storage/PurgeDatabaseAndroidTest.kt

```diff
@@ -79,7 +79,7 @@ class PurgeDatabaseAndroidTest {
     assertThat(patientDao.patientProfileImmediate(deletedButUnsyncedPatientProfile.patientUuid)).isEqualTo(deletedButUnsyncedPatientProfile)
 
     // when
-    appDatabase.purge(testUserClock)
+    appDatabase.purge(Instant.now(testUserClock))
```

Comment:
If we do this, tests are not deterministic anymore. For example, let's say we have set the `retainUntil` to sometime in future in a test verifying it doesn't get deleted. Once `Instant.now` exceeds that, those tests will start failing. Instead, we should use `Instant.parse("2018-01-01T00:00:00Z")` to set a more deterministic value that we can control and stays consistent.

Reply:
Oh yes, my mistake. Will update this.